Makefile

```diff
@@ -142,6 +142,18 @@ frontend-lint: node_modules/.uptodate
 bddtests: python
 	@tox -qe bddtests
 
+requirements/%.txt: requirements/%.in
+	@tox -qe $(subst requirements,dev,$(basename $(notdir $@))) --run-command 'pip-compile --allow-unsafe --quiet $(PIP_COMPILE_FLAGS) $<'
```

Comment:
```suggestion
	@tox -qe $(subst requirements,dev,$(basename $(notdir $@))) --run-command 'pip-compile --allow-unsafe --quiet $(args) $<'
```

The interface for using this isn't going to be very nice, but I think it would be better to follow the pattern of `make dockercompose` and use `args` for this sort of thing. I don't think we need to namespace the args command as it's implicit from which make command we call.

This is some dense bash stuff here, which would probably benefit from a comment.

Reply:
Yeah fair enough, `$(args)` is kind of our own made-up pattern instead of general Make conventions, on the other hand it's shorter and it's what we do in our other make commands, will do.

Regarding namespacing of `args`--the way we use it it's actually a target-specific variable currently so it is sort of localized.

It's true that the UI for this is never going to be nice. FWIW the cookiecutter generates a `HACKING.md` file that includes copy-pastable commands for all of these.